content/en/blog/2024/otel-generative-ai/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: OpenTelemetry for Generative AI
+linkTitle: OpenTelemetry for Generative AI
```

Comment:
Drop the stuttered `linkTitle` or shorten it, e.g.:

```suggestion
title: OpenTelemetry for Generative AI
linkTitle: OTel for Generative AI
```

My preference would be to drop it.